presto-main/src/main/java/com/facebook/presto/execution/QueryPreparer.java

```diff
@@ -54,6 +60,12 @@ public PreparedQuery prepareQuery(Session session, String query, WarningCollecto
             throws ParsingException, PrestoException, SemanticException
     {
         Statement wrappedStatement = sqlParser.createStatement(query, createParsingOptions(session, warningCollector));
+        if (SystemSessionProperties.getWarningHandlingLevel(session) == WarningHandlingLevel.AS_ERROR) {
```

Comment:
Static import.

Reply:
Should you check whether there are warnings before throwing exception?

Reply:
Yes I should be checking. I will fix that. Whats the issue with the static import?

Reply:
It's just convention. We always do static import unless the function name / constant by itself is too confusing.